benchmarks/online-shop/update_remote_files.sh

```diff
@@ -36,6 +36,16 @@ done
 popd
 
 
+
+##################
+## Manual fixes ##
+
+# No support for google cloud debugger at the moment for arm.
+sed -i 's|google-python-cloud-debugger|#google-python-cloud-debugger|g' recommendationservice/requirements.txt
```

Comment:
You should change make module import conditional on the architecture. Please see [this](https://docs.python.org/3/library/platform.html).

Reply:
I removed the cloud-debugger now entirely. We don't need it.
Though, I would leave this (uncommented), that we remember this once we upgrade to a new version